src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/Completions.test.ts

```diff
@@ -98,11 +95,25 @@ suite('Completions', () => {
             'vscode.executeCompletionItemProvider',
             cshtmlDoc.uri,
             new vscode.Position(lastLine.line, 1));
-        const matchingCompletions = completions!.items
-            .filter(item => (typeof item.insertText === 'string') && item.insertText.startsWith('DateTime'))
-            .map(item => item.insertText as string);
 
-        assert.deepEqual(matchingCompletions, ['DateTime', 'DateTimeKind', 'DateTimeOffset']);
+        assertHasCompletion(completions, 'DateTime');
+        assertHasCompletion(completions, 'DateTimeKind');
+        assertHasCompletion(completions, 'DateTimeOffset');
+    });
+
+    test('Can perform Completions on file with "#" in the name', async () => {
```

Comment:
Not sure I would have added a functional test for this given the cost of having them but don't feel strongly